x/pki/keeper/all_certificates_by_subject_key_id.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/zigbee-alliance/distributed-compliance-ledger/x/pki/types"
 )
 
```

Comment:
Did we update all test cases from https://github.com/zigbee-alliance/distributed-compliance-ledger/blob/master/x/pki/tests/test-design.md? Please update description of the PR.
Please mark the tests in test design that have been refactored and check all requires states.